prompt-service/src/unstract/prompt_service/plugins/table_extractor/src/processor/post_processor.py

```
@@ -0,0 +1,245 @@
+            return result.stdout
+        except subprocess.CalledProcessError as e:
+            logger.error(f"Script execution failed: {e.stderr}")
+            logger.error(f"Command failed with exit code {e.returncode}:")
+            raise PluginException(
+                f"Unexpected error occured while post processing. {str(e)}",
+            ) from e
+        except FileNotFoundError:
+            logger.error("The command could not be found.")
+        except TimeoutError:
+            logger.error("The command timed out.")
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)